MSAL/src/public/MSAL.h

```diff
@@ -53,3 +53,11 @@ FOUNDATION_EXPORT const unsigned char MSAL__Framework_VersionString[];
 #import <MSAL/MSALB2CAuthority.h>
 #import <MSAL/MSALADFSAuthority.h>
 #import <MSAL/MSALPublicClientStatusNotifications.h>
+#import <MSAL/MSALSilentTokenParameters.h>
+#import <MSAL/MSALInteractiveTokenParameters.h>
+#import <MSAL/MSALTokenParameters.h>
+#import <MSAL/MSALClaimsRequest.h>
+#import <MSAL/MSALIndividualClaimRequest.h>
+#import <MSAL/MSALIndividualClaimRequestAdditionalInfo.h>
+#import <MSAL/MSALJsonSerializable.h>
```

Comment:
Do we need to expose MSALJsonSerializable and deserializable?

Reply:
Yes, deserializable is definitely needed (allows a developer to init claim request with json string). Serializable could be private but it would be nice to have it public too.